include/nccl_ofi_scheduler.h

```diff
@@ -127,9 +126,11 @@ int nccl_net_ofi_threshold_scheduler_init(int num_rails,
  * A mininal stripe size `max_stripe_size' is calculated (multiple of
  * `align') that is sufficient to assign the whole message. Rails are
  * filled from low id to large id. The last rail may get assigned less
- * data.
+ * data. The number of rails are calculated based on the ratio of
+ * (`data_size` / `min_stripe_size`)
  */
-void nccl_net_ofi_set_multiplexing_schedule(size_t size,
+int nccl_net_ofi_set_schedule_by_threshold(nccl_net_ofi_threshold_scheduler_t *scheduler,
```

Comment:
I know it was like this before your changes, but I am wondering why we have this function in the header file if it is only used internally in the scheduler. If it doesn't need to be here, remove also the `nccl_net_ofi_` prefix, since we only use it for non-internal functions.

Reply:
Done